lint/rule_panel_rate_interval.go

```diff
@@ -23,7 +23,7 @@ func NewPanelRateIntervalRule() *PanelRuleFunc {
 			}
 
 			switch p.Type {
-			case "singlestat", "graph", "table":
```

Comment:
Can we update this function to use `panelHasQueries`?

Reply:
Sure, I proposed a new version below.
I just wonder if it is better like I did with an early return or if you prefer to protect the `for _, target := range p.Targets {...}` with an `if panelHasQueries(p)`